### PR TITLE
[ci] Disable a few vulkan flaky tests.

### PR DESCRIPTION
**Pull request by ailzhang**
Our GPU CI is pretty flaky these days, I happen to see a few failed tests on my linux workstation which I think are pretty relevant. I'm going to disable these first to make sure CI is back to normal state. 
Screen Shot 2021-09-14 at 9 40 46 AM

### CODE REVIEW

File: tests/python/test_ad_basics.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def test_poly():
 
 
 @if_has_autograd
+@ti.test(exclude=[ti.vulkan])
 def test_trigonometric():
     grad_test(lambda x: ti.tanh(x), lambda x: np.tanh(x))
     grad_test(lambda x: ti.sin(x), lambda x: np.sin(x))
```
Comment (g1n0st) [inline tests/python/test_ad_basics.py:82]: `ti.vulkan` seems not support autograd yet...

File: tests/python/test_clear_all_gradients.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 
 @ti.test()
+@ti.test(exclude=[ti.vulkan])
```
Comment (g1n0st) [inline tests/python/test_clear_all_gradients.py:5]: ditto

File: tests/python/test_element_wise.py [hunk 3]
```diff
@@ -243,6 +243,7 @@ def func():
 
 
 @ti.test()
+@ti.test(exclude=[ti.vulkan])
```
Comment (g1n0st) [inline tests/python/test_element_wise.py:246]: This might be the accuracy problem, rel=1e-5 probably not enough.

**Review by g1n0st — APPROVED** [review]
LGTM!

File: tests/python/test_ad_basics.py (hunk 1 above)
Comment (ailzhang) [reply tests/python/test_ad_basics.py:82]: This seems to be running fine on vulkan, except for an accuracy problem too :P 
`FAILED tests/python/test_ad_basics.py::test_trigonometric - assert 1.3344652652740479 == 1.3346064424467945 ± 1...`

Comment (g1n0st) [reply tests/python/test_ad_basics.py:82]: But it's also need to note that the accuracy in Vulkan backend seems lower than other backends...

File: tests/python/test_clear_all_gradients.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 
 @ti.test()
```
Comment (strongoier) [inline tests/python/test_clear_all_gradients.py:4]:
```suggestion
```
You don't have to repeat `ti.test()` :-)

File: tests/python/test_element_wise.py [hunk 5]
```diff
@@ -243,6 +243,7 @@ def func():
 
 
 @ti.test()
```
Comment (strongoier) [inline tests/python/test_element_wise.py:245]:
```suggestion
```
ditto

File: tests/python/test_mpm88.py [hunk 6]
```diff
@@ -127,6 +127,7 @@ def test_mpm88_async():
 
 
 @ti.test()
```
Comment (strongoier) [inline tests/python/test_mpm88.py:129]:
```suggestion
```
ditto

File: tests/python/test_struct.py [hunk 7]
```diff
@@ -63,6 +63,7 @@ def test_linear_nested_aos():
 
 
 @ti.test()
```
Comment (strongoier) [inline tests/python/test_struct.py:65]:
```suggestion
```
ditto

File: tests/python/test_clear_all_gradients.py (hunk 4 above)
Comment (ailzhang) [reply tests/python/test_clear_all_gradients.py:4]: oops nice catch! thanks!

**Pull request merged**